hydra-node/test/Hydra/Chain/Direct/WalletSpec.hs

```diff
@@ -196,6 +198,24 @@ prop_seenInputsAreConsumed =
 -- coverFee
 --
 
+prop_setsMinUTxOValue :: Property
+prop_setsMinUTxOValue =
+  forAllBlind (resize 0 genLedgerTx) $ \tx ->
+    forAllBlind (reasonablySized $ genOutputsForInputs tx) $ \lookupUTxO ->
+      forAllBlind (reasonablySized genUTxO) $ \walletUTxO ->
+        -- Generate another txOut and make it deliberately "under-valued"
+        forAll arbitrary $ \txOut -> do
+          let txOutWithoutADA = txOut & coinTxOutL .~ mempty
+          let newTx = tx & bodyTxL . outputsTxBodyL <>~ StrictSeq.singleton txOutWithoutADA
+          case coverFee_ Fixture.pparams Fixture.systemStart Fixture.epochInfo lookupUTxO walletUTxO newTx of
```

Comment:
With the `Fixture.` prefix, this looks cluttered, perhaps a good time to group those parameters together?